README.md

```diff
@@ -5,18 +5,66 @@ This repository contains specifications defined by the BioImage.IO community. Th
 
 All the BioImage.IO-compatible RDF must fulfill the following rules:
  * Must be a YAML file encoded as UTF-8;
- * The RDF file extension must be `.yaml` (instead of `.yml`)
- * The RDF file can be saved in a folder(or virtual folder) or in a zip package, the following additional rules must apply:
+ * The RDF file extension must be `.yaml` (not `.yml`)
+ * The RDF file can be saved in a folder (or virtual folder) or in a zip package, the following additional rules must apply:
    1. When stored in a local file system folder, github repo, zenodo deposition, blob storage virtual folder or similar kind, the RDF file name should match the pattern of `*.rdf.yaml`, for example `my-model.rdf.yaml`.
    2. When the RDF file and other files are zipped into a RDF package, it must be named as `rdf.yaml`.
 
-As a general guideline, please follow the model RDF spec to describe AI models and use the generic RDF spec for other resource types including `dataset`, `notebook`, `application`. You will find more details about these two specifications in the following sections. Please also note that the best way to check whether your RDF file is BioImage.IO-complaint is to run the BioImage.IO Vallidator against it.
+As a general guideline, please follow the model RDF spec to describe AI models and use the generic RDF spec for other resource types including `dataset`, `notebook`, `application`. You will find more details about these two specifications in the following sections. Please also note that the best way to check whether your RDF file is BioImage.IO-complaint is to run the BioImage.IO Validator against it.
 
-## [Resource Description File Specification](./bioimageio_rdf_spec.md)
+## [Resource Description File Specification](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_latest.md)
 
-A BioImage.IO-compatible Resource Description File (RDF) is a YAML file with a set of specifically defined fields. You can find detaialed field definition and examples here in the [generic RDF spec](./bioimageio_rdf_spec.md).
+A BioImage.IO-compatible Resource Description File (RDF) is a YAML file with a set of specifically defined fields. 
 
-## [Model Resource Description File Specification](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/bioimageio_model_spec.md)
+You can find detailed field definitions here: 
+   - [generic RDF spec 0.3](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_3.md)
+
+The specifications are also available as json schemas: 
+   - [generic RDF spec 0.3 (json schema)](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_3.json)
+
+### Describing applications
```

Comment:
I think it might be better to move the examples of generic RDF to a separate md file, to make the whole readme shorter.

Reply:
I would prefer pointing to actual, working examples, like for the non generic rdf for a model:
https://github.com/CellProfiling/HPA-model-zoo/blob/2f668d87defddc6c7cd156259a8be4146b665e72/manifest.bioimage.io.yaml#L33-L59 

Would you have a pointer to the kaibu spec for me?

Reply:
That make sense, however, I would keep the text as is for now, there are changes we need to sort out for using zenodo to also store generic RDFs.

Reply:
ok, but then let's not worry about a lengthy readme.md for now...